modules/kie-cloud-operator/install

```diff
@@ -1,9 +1,5 @@
 #!/bin/sh
 
-INSTALL_PKGS="go-toolset-1.10"
```

Comment:
This var is used to remove go later in the script.

Reply:
Sry, missed that.

Yes, the image size is under 400mb:
2019-04-10 09:51:11,496 root         INFO     Original image size: 922.69 MB
2019-04-10 09:51:11,496 root         INFO     Squashed image size: 376.81 MB


Just to be sure, when squashing do you select a layer or squash all layers?

Reply:
whatever is default in brew